backend/django/core/views/api_annotate.py

```diff
@@ -546,6 +552,12 @@ def data_admin_table(request, project_pk):
     project = Project.objects.get(pk=project_pk)
     queue = Queue.objects.get(project=project, type="admin")
 
+    messages = list(
```

Comment:
I'm wondering if we can just move this into the data serializer like the metadata? If there's some reason it's not straightforward that's fine, but I'm worried this might get slow later.

Reply:
The reason I was thinking of keeping this separate was because the user only needs to see this data on the Skipped page. On annotate, history, etc. there's no need to fetch that data (it also won't even exist) unlike the metadata which is present on every page's cards. Thoughts?

Reply:
Oh I was thinking we could make a serializer just for the skipped page.

Reply:
I think for now this is fine.